include/mgba-util/common.h

```diff
@@ -31,7 +31,7 @@ CXX_GUARD_START
 #include <time.h>
 
 #ifdef _WIN32
-// WinSock2 gets very angry if it's included too late
```

Comment:
Does the PS2 version of GCC get angry at apostrophes in comments or something?

Reply:
Yes, it generate compiling errors, I don't know why

Reply:
They are C++ comments, not C comments. PS2 toolchain is likely using a stone age version of GCC.

Reply:
I figured that might be the case. It seems to set std=c99 just fine though, so idk.